toc.go

```diff
@@ -96,7 +96,8 @@ type indexTOC struct {
 	contentChecksums simpleSection
 	runeDocSections  simpleSection
 
-	repos simpleSection
+	repos          simpleSection
```

Comment:
Should we bump the index `FeatureVersion` or `FormatVersion` here? (I'm not 100% on which one 😊 ) That way, far in the future once we've dropped support for older versions, we can know the bitmap is always there, and simplify the logic?

Reply:
Sounds good to do it in a separate PR. Also let's catch up about `FeatureVersion`/ `FormatVersion` more generally, I'd like to understand how to simplify the model.